editor/src/components/canvas/canvas-strategies/strategies/flex-reorder-strategy.spec.browser2.tsx

```diff
@@ -324,4 +418,255 @@ describe('Flex Reorder Strategy', () => {
       </div>`),
     )
   })
+
+  xdescribe('projects with fragments, with fragments support enabled', () => {
```

Comment:
is this test disabled for a reason? or just leftover?

Reply:
This was done so that we don't have to get the fragments work fully functional (with the feature switch on) before merging it as these tests do not all pass currently.